webtau-docs/znai/GraphQL/customized-graphql-urls.md

```diff
@@ -0,0 +1,20 @@
+# Customizing the GraphQL Endpoint
+
+By default, Webtau assumes that all requests are made to a `/graphql` endpoint. 
```

Comment:
We should standardise on webtau vs. Webtau vs. WebTau.  Pretty sure I've seen all three in our docs 😁

Reply:
Very true. 
The master pom says `<description>WebTau - web test automation. REST/GraphQL/DB/CLI/UI testing framework.</description>` , so maybe we go with that? Up to @MykolaGolubyev 

I can do a global search&replace in .md files in a separate PR.

Reply:
Fair point. The reason for PascalCase in description is to get the abbreviation across. After that I say we use `webtau` everywhere. Maybe even replace the WebTau in description. 
Would love a PR with this fix!